sqlglot/dialects/presto.py

```diff
@@ -626,3 +627,30 @@ def create_sql(self, expression: exp.Create) -> str:
             if kind == "VIEW" and schema.expressions:
                 expression.this.set("expressions", None)
             return super().create_sql(expression)
+
+        def delete_sql(self, expression: exp.Delete) -> str:
+            """
+            Presto only support DELETE FROM a table, without alias, so we need to remove the unnecessary parts.
+            If the original DELETE statement contains more than one table to be deleted, we can't do anything.
+            """
+
+            tables = expression.args.get("tables")
+            if tables:
+                if len(tables) > 1:
+                    return super().delete_sql(expression)
+                else:
+                    target_table = tables[0]
+                    del expression.args["tables"]
+                    expression.set("this", target_table)
+
+            table = expression.this
+            if isinstance(table, exp.Table):
+                table_alias = table.find(exp.TableAlias)
+                if table_alias:
+                    table_alias.pop()
+
+            qualified_expression = expression.transform(unqualify_columns)
+            if isinstance(qualified_expression, exp.Delete):
+                return super().delete_sql(qualified_expression)
+
+            return super().delete_sql(expression)
```

Comment:
I think we can simplify this section?
```suggestion
            return super().delete_sql(expression.transform(unqualify_columns))
```

Reply:
If you do that, you would fail the mypy checkstyle

Reply:
We shouldn't avoid simplifying the logic to satisfy mypy. In this case we obviously get back a `DELETE` from the transform, so we should either `# type: ignore` it or `t.cast(exp.Delete, ...)`.